client/landing/stepper/declarative-flow/hundred-year-domain-transfer.ts

```diff
@@ -20,16 +21,28 @@ const hundredYearDomainTransfer: Flow = {
 	// Always start on the "domains" step, as we don't need what comes before it for the 100-year domain transfer flow.
 	// It doesn't make sense to show it as it contains additional cluttering information that doesn't matter
 	// specifically under the 100-year domain context - they'll also come from a different entry point.
-	useSteps() {
-		const transferSteps = domainTransfer.useSteps();
-		const domainsStepIndex = transferSteps.findIndex( ( step ) => step.slug === 'domains' );
-		return [
-			transferSteps[ domainsStepIndex ],
-			...stepsWithRequiredLogin( transferSteps.slice( domainsStepIndex ) ),
-		];
+	async initialize() {
+		const transferSteps = await domainTransfer.initialize();
+
+		if ( transferSteps === false ) {
+			// Forwarding returning the `false` value, as it's a signal to kill the app.
+			return false;
+		}
+
+		const domainsStepIndex = transferSteps.findIndex(
+			( step ) => step.slug === STEPS.DOMAIN_TRANSFER_DOMAINS.slug
+		);
+
+		if ( domainsStepIndex > -1 ) {
+			// If the needed step was not found, abort the flow.
+			return false;
+		}
+
+		return [ ...stepsWithRequiredLogin( transferSteps.slice( domainsStepIndex ) ) ];
 	},
 
 	// Always allow the user to proceed with the transfer, as we assert that the user is logged in later.
+	// TODO: why?
 	useAssertConditions(): AssertConditionResult {
 		return { state: AssertConditionState.SUCCESS };
 	},
```

Comment:
This was probably here to override the `useAssertConditions` function from the `domainTransfer` flow, and therefore I believe it's safe to delete. Especially given that now every step in the flow is wrapped in `stepsWithRequiredLogin`?

Reply:
That's correct.